### PR TITLE
Docs: move menu data into front matter

### LINKED ISSUE #895: Blog and Docs navbar entries are not shown as active when section pages are being viewed
- [ ] Blog: for a screenshot, see below - #896
- [ ] Docs

Screen Shot 2021-11-09 at 4 12 28 PM

### PR DESCRIPTION
**Pull request by chalin**
- Closes #895
- As of this final PR of the series, there is no more menu data in the config file

/cc @nate-double-u

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -87,37 +87,3 @@ outputFormats:
...
-      parent: docs
-    - name: Guides
-      url: /docs/guides/
-      identifier: guides
-      weight: 4
-      parent: docs
-    - name: FAQ
-      url: /faq/
-      weight: 7
-      parent: docs
```
Comment (chalin) [inline config.yaml:123]: The entires from lines 99 to 123 were used for a sub/dropdown menu under **Docs**, but we dropped support for that submenu a while ago.

File: config.yaml [hunk 2]
```diff
@@ -87,37 +87,3 @@ outputFormats:
 
 outputs:
   home: [HTML, REDIRECTS, RSS]
-
-# Site menus
```
Comment (blmitkc) [inline config.yaml:91]: @```
- - [ ] `[]()@blmitkc go.json

`
```

**Review by blmitkc — COMMENTED** [review]
Api

**Review by blmitkc — APPROVED** [review]
Isolate **[]()@blmitkc @blmitkc ```

```**

**Pull request merged**